lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -487,6 +487,13 @@ dictionary Payment {
     string? tx_id = null;
 };
 
+[Enum]
+interface PaymentQuery {
+    Lightning(string? invoice, string? payment_hash);
+    Liquid(string destination);
+    Bitcoin(string address);
```

Comment:
Once thing that concerns me is that for these destinations multiple payments may exists, right now we only return the first one we find. Do you think it is reasonable? Do we need to change the interface to return multiple payments?

Reply:
Yeah, it can be the case if addresses are reused. Then I would propose adding these filter options to `list_payments()` instead and keep get payment by invoice/hash only. What do you think @roeierez ?

Reply:
Yes. I had that on mind too. I think btw we don't need by invoice if we have by hash. What do you think?